src/fabric-1.19.2/generator.yaml

```diff
@@ -65,8 +65,8 @@ base_templates:
 - template: modbase/mixins.json.ftl
   writer: file
   name: "@RESROOT/@modid.mixins.json"
-  condition: ${w.hasToolsOfType("Fishing rod")}
-  deleteWhenConditionFalse: true
+  condition: ${w.hasElementsOfBaseType("item")}
+  deleteWhenConditionFalse: false
```

Comment:
It is not needed when you want it false.

Reply:
It should be true, not false